src/app/providers.tsx

```
@@ -0,0 +1,30 @@
+"use client";
+
+import { ClerkProvider } from "@clerk/nextjs";
+import { TRPCReactProvider } from "@/trpc/client";
+import { useCartSync } from "@/features/cart/hooks/use-cart-sync";
+
+function CartSync() {
+  const {} = useCartSync();
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)